### PR TITLE
Change image and text NOAA -> Bio-Oracle

### PR DESCRIPTION
**Pull request by salvafern**

### CODE REVIEW

**Review by bart-v — CHANGES_REQUESTED** [review]
1. I think the img src should be "erddap/images/bio_oracle_thumbs.png"
2. Normally, you also add a link back to bio-oracle.org, just like NOAA does it?

File: erddap/content/datasets_template.xml [hunk 1]
```diff
@@ -21,7 +21,12 @@
     <updateMaxEvents></updateMaxEvents>                               <!-- default=10 -->
     <!-- The defaults for the following tags are in messages.xml. -->
     <startHeadHtml5></startHeadHtml5>
-    <startBodyHtml5></startBodyHtml5>                                 <!-- This is often customized. -->
+    <startBodyHtml5>
+		<a href="https://bio-oracle.org/">
```
Comment (vinisalazar) [inline erddap/content/datasets_template.xml:25]: Perhaps link to the ERDDAP itself rather than the BO website?

Comment (bart-v) [reply erddap/content/datasets_template.xml:25]: I think it's OK like this

File: erddap/content/datasets_template.xml [hunk 2]
```diff
@@ -21,7 +21,12 @@
...
+			<img src="tomcat/content/erddap/bio_oracle_thumbs.png" alt="Bio-Oracle">
```
Comment (bart-v) [inline erddap/content/datasets_template.xml]: The image URL path does not start with "tomcat/(...)"
But with "erddap/(...)"

**Comment by Fred-Leclercq** [comment]
You might need to change the path to the image:
```
<img src="tomcat/content/erddap/bio_oracle_thumbs.png" alt="Bio-Oracle"> 
=>
<img src="&erddapUrl;/images/bio_oracle_thumbs.png" alt="Bio-Oracle">
```

**Comment by salvafern** [comment]
and the image goes into tomcat/content/erddap/ or tomcat/content/erddap/images/ ?

**Comment by bart-v** [comment]
The latter .../images/

**Comment by salvafern** [comment]
ok to merge?

**Review by bart-v — APPROVED** [review]

**Comment by bart-v** [comment]
Yes

**Pull request merged**